VMBackup/main/Utils/HandlerUtil.py

```
@@ -140,18 +142,24 @@ def exit_if_same_seq(self):
             sys.exit(0)
 
     def log(self, message,level='Info'):
-        WriteLog = self.get_value_from_configfile('WriteLog')
-        if (WriteLog == None or WriteLog == 'True'):
-            if sys.version_info > (3,):
-                if self.logging_file is not None:
-                    self.log_py3(message)
```

try-catch to ignore error in tracing